test/modules/sparlql-query-service.test.js

```diff
@@ -0,0 +1,178 @@
+const {
+    describe,
+    it,
+    before,
+    after,
+} = require('mocha');
+const chai = require('chai');
+
+const {
+    assert,
+    expect,
+} = chai;
+
+const Sparql = require('../../external/sparqlquery-service');
+const Logger = require('../../modules/logger/logger');
+
+const config = { url: 'http://localhost:9999/blazegraph/namespace/kb/sparql' };
```

Comment:
Let's parametrize this part and take from config, also can we make this triple-store independent, to work the same with GraphDB and Blazegraph

Reply:
Resolved. We now load the sparqlendpoint url from config.